scripts/version

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env bash
+
+# Usage: version <bump>
+#   bump: [possible values: major, minor, patch, release, rc, beta, alpha]
+
+# Depends
+if ! (command -v cargo-set-version > /dev/null &&
+      command -v jq > /dev/null &&
+      command -V yq > /dev/null)
```

Comment:
Is the differing capitalization of `-v` intentional?

Reply:
Not intentional, thanks for the catch! funnily enough, it does the same thing, but I'll still fix

Reply:
1f1a0d1